docs/reference/glossary.md

```diff
@@ -483,203 +404,216 @@ Changes to the world are made by [workers](#worker). Each worker has a view onto
+
+> **Related:**
 >
 > * [Worker attributes](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#worker-attribute)
 > * [Bridge configuration](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/worker-configuration/bridge-config#worker-attribute-sets)
 
-### Worker flags
+## Worker flags
 
-A worker flag is a key-value pair that workers can access during runtime. Worker flags can be set in their [launch configuration](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/reference/file-formats/launch-config) or added/changed/removed at runtime through the SpatialOS console. [This SpatialOS documentation](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/worker-configuration/worker-flags#worker-flags) describes how to define and change worker flags.
+A worker flag is a key-value pair that workers can access during runtime. 
 
-### Worker SDK
+Worker flags can be set in their [launch configuration](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/reference/file-formats/launch-config). 
 
-The GDK is built on top of a specialized version of the C# Worker SDK. This specialized C# Worker SDK is itself built on top of the [C API](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/capi/introduction). The Worker SDK handles the implementation details of communicating to the SpatialOS runtime and provides a serialization library.
+Worker flags can be modified at runtime through:
 
-> Related:
+* The [`spatial` CLI](#spatial-command-line-tool-cli) for local and cloud deployments.
+* The [SpatialOS Console](#spatialos-console) for cloud deployments.
+* The Platform SDK for cloud deployments.
+
+> **Related:**
 >
-> * [Worker SDK](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#worker-sdk)
+> * [Worker flag documentation](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/worker-configuration/worker-flags#worker-flags)
+> * [`spatial` CLI documentation](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/spatialos-cli-introduction)
+> * [Platform SDK API reference](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/platform-sdk/reference/home)
 
-### Worker Origin
+## Worker SDK
 
-The worker origin allows you to translate the origin of all SpatialOS entities checkout out by a given worker. When connecting multiple workers in the same scene, you can use this origin as an offset to ensure no unwanted physical interactions between those game objects occur.
+The GDK is built on top of a specialized version of the C# Worker SDK. This specialized C# Worker SDK is itself built on top of the [C API](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/capi/introduction). 
```

Comment:
link is broken

Reply:
also do we need to say it's a specialized version instead of just linking to it

Reply:
I guess its to steer people away from looking at the C# Worker SDK docs and getting confused?

Reply:
Fixed the link

Reply:
ah yeah. I thought we had added documentation for that, but doesn't look like it. In that case that's fine